source/common/router/rds_impl.cc

```diff
@@ -146,7 +145,7 @@ void RdsRouteConfigSubscription::onConfigUpdate(
     *unwrapped_resource.Add() = added_resources[0].resource();
     onConfigUpdate(unwrapped_resource, added_resources[0].version());
   } else {
-    onConfigUpdate({}, system_version_info);
```

Comment:
Why do we need to do a `onConfigUpdate` at all if there are no resources?

Reply:
Oh right, because we're leaving the ability to remove resources as a todo. Great, that makes this much simpler; no need to worry about the weirdness of "". Thanks!